src/status_im/utils/keychain.cljs

```diff
@@ -14,9 +14,12 @@
            (when reject
              (reject))
            (let [encryption-key (.parse js/JSON (.-password res))]
-             (log/debug "Found existing encryption key!")
-             (re-frame/dispatch [:got-encryption-key {:encryption-key encryption-key
-                                                      :callback       resolve}])))))
+             (if (= (.-length encryption-key) 0)
```

Comment:
I would also check "right size — all zeroes" case.

Reply:
@mandrigin yes, I just would like to confirm what is the problem ( is just a wip build cannot replicate locally), but I will add more checks after, we also need to panic at the very minimum, or handle in some way otherwise, rather than just silently leaving the db unencrypted.